api/communication-api/src/main/java/com/cognifide/aet/communication/api/metadata/Statistics.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Automated Exploratory Tests
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.aet.communication.api.metadata;
+
+import java.io.Serializable;
+
+public class Statistics implements Serializable {
```

Comment:
Yes, my idea was to add more data here in the future, e.g. start/end time if required, maybe some wait/sleep time required to pass.
I'd like to have statistics of how long AET really needed to perform a test (how long loading/sleep took vs real test time).